src/gui/org/deidentifier/arx/gui/view/impl/define/ViewAttributeTransformation.java

```diff
@@ -150,7 +151,7 @@ public ViewAttributeTransformation(final Composite parent,
         cmbType.addSelectionListener(new SelectionAdapter() {
             @Override
             public void widgetSelected(final SelectionEvent arg0) {
-                actionAttributeTypeChanged();
+                actionAttributeTypeChanged(attribute, COMBO1_TYPES[cmbType.getSelectionIndex()]);
```

Comment:
Your are now lacking the sanity checks that were implemented previously. E.g. if ((cmbType.getSelectionIndex() != -1) && (attribute != null)) {